lib_nn/api/vpu.hpp

```diff
@@ -0,0 +1,53 @@
+#include <cstdint>
+#include <ostream>
+#include <cstring>
+
+#ifndef VPU_HPP_
```

Comment:
It might be good to change the include guard to something larger such as LIB_NN_API_VPU_HPP_ to indicate the full path, in case we include another vpu.hpp somewhere else.

Reply:
good plan